cmd/ltctl/comparison.go

```diff
@@ -161,16 +161,32 @@ func RunComparisonCmdF(cmd *cobra.Command, args []string) error {
 		if err := writeReports(output.Results, outputPath); err != nil {
 			return fmt.Errorf("failed to write reports: %w", err)
 		}
-		printResults(output.Results)
-	}
 
-	if archive {
-		if err := createArchive(outputPath, archivePath); err != nil {
-			return fmt.Errorf("failed to create archive: %w", err)
+		//create the file
+		resultsFile, errResult := os.Create(filepath.Join(outputPath, "results.txt"))
+		if errResult != nil {
+			fmt.Println("Error:", errResult)
+			return fmt.Errorf("failed to create file: %w", err)
+		}
```

Comment:
We can remove this `fmt.Println` line as well. Also, it seems we were not using the correct error in the `fmt.Errorf` call:
```suggestion
		//create the file
		resultsFile, errResult := os.Create(filepath.Join(outputPath, "results.txt"))
		if errResult != nil {
			return fmt.Errorf("failed to create file: %w", errResult)
		}
```